app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -1,5 +1,103 @@
 module Api
   class ConversionHostsController < BaseController
     include Subcollections::Tags
+
+    # Whitelist of valid resource types
+    VALID_TYPES = {
+      'ManageIQ::Providers::Openstack::CloudManager::Vm' => ManageIQ::Providers::Openstack::CloudManager::Vm,
+      'ManageIQ::Providers::Redhat::InfraManager::Host'  => ManageIQ::Providers::Redhat::InfraManager::Host
+    }
+
+    # Create a conversion host and enable it. This operation will run as an
+    # MiqTask.
+    #
+    # Both the 'resource_type' and 'resource_id' are mandatory arguments,
+    # and the 'resource_type' must be either 'Host' or 'VmOrTemplate'.
+    #
+    # You may optionally pass in 'param_v2v_vddk_package_url' or 'auth_user'
+    # arguments as well.
+    #
+    # POST /api/conversion_hosts {
+    #   "name": "some_name",
+    #   "resource_type": "Host",
```

Comment:
The `"Host"` in this example is not correct, right? It needs to be either of the types listed in VALID_TYPES.

Reply:
Oops, thought I fixed that. Thanks, updated.